public/app.js

```diff
@@ -1,13 +1,38 @@
-<!DOCTYPE html>
-
-<html lang="en">
-<head>
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-
-  <title>Project Title</title>
-  <link rel="stylesheet" href="app.css">
-</head>
-<body>
-</body>
-</html>
+(function() {
+  var timer = document.querySelector('.time');
+  var colorGrad = document.querySelector('.color');
+  var bar = document.querySelector('.timer__bar');
+  var backgroundColor = document.querySelector('.clock');
+
+  window.setInterval(() => {
+    var currentTime = new Date();
+    var nowHour = padZeros(currentTime.getHours());
+    var nowMin = padZeros(currentTime.getMinutes());
+    var nowSec = padZeros(currentTime.getSeconds());
+    var now = `${nowHour}:${nowMin}:${nowSec}`;
+    timer.innerHTML = now;
+
+    var secColor = toColorRange(nowSec, 60);
+    var minColor = toColorRange(nowMin, 60);
+    var hourColor = toColorRange(nowHour, 60);
+
+    var colorVal = `rgb(${secColor}, ${minColor}, ${hourColor})`;
+
+    backgroundColor.style.background = colorVal;
+    colorGrad.innerHTML = colorVal;
+    bar.style.width = (nowSec * 5 + 'rem');
```

Comment:
This value is far too large and leading to the bar almost always taking up the full screen. Ideally this would be in a percentage value and then the parent element could be set to a max width to be able to get a solid result every time.